test/SlowTests/Server/RecordingTransactionOperationsMergerTests.cs

```diff
@@ -0,0 +1,500 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using FastTests;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Raven.Client;
+using Raven.Client.Documents.Commands;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Documents.Queries;
+using Raven.Client.Documents.Session;
+using Raven.Client.Documents.Smuggler;
+using Raven.Server.Documents;
+using Raven.Server.ServerWide.Context;
+using Raven.Server.Smuggler.Documents;
+using Raven.Server.Smuggler.Documents.Data;
+using Raven.Tests.Core.Utils.Entities;
+using Xunit;
+
+namespace SlowTests.Server
+{
+    public class RecordingTransactionOperationsMergerTests : RavenTestBase
+    {
+        private const string AgeKey = "age";
+
+        [Fact]
+        public void JustRecord()
+        {
+            var dateString = DateTime.Now.ToString("G").Replace(" ", "").Replace(":", "").Replace(@"/", "");
```

Comment:
FYI, you can format everything in `.ToString()`

Reply:
done